pkg/splunk/enterprise/configuration.go

```
@@ -522,9 +528,57 @@ func addStorageVolumes(cr splcommon.MetaObject, spec *enterpriseApi.CommonSplunk
 		}
 	}
 
+	// Add Splunk Probe config map
+	probeConfigMap := getProbeConfigMap(ctx, client, cr)
+	if probeConfigMap == nil {
+		scopedLog.Error(nil, "Unable to get probeConfigMap")
+	}
+	addProbeConfigMapVolume(probeConfigMap, statefulSet)
 	return nil
 }
 
+func getProbeConfigMap(ctx context.Context, client splcommon.ControllerClient, cr splcommon.MetaObject) *corev1.ConfigMap {
+
+	configMap := corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      GetProbeConfigMapName(cr.GetNamespace()),
+			Namespace: cr.GetNamespace(),
+		},
+	}
+
+	// Add readiness script to config map
+	data, err := ReadFile(ctx, GetReadinessScriptLocation())
+	if err != nil {
+		return nil
```

Can you also propagate the error ?

---

+1